projects/hello-cmake-package/CMakeLists.txt

```
@@ -0,0 +1,127 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
```

```suggestion
cmake_minimum_required(VERSION 3.14...3.19)
```

I don't think CMake 2.6 is common enough to worry about adding the FATAL_ERROR anymore. Need to make sure 3.20 is fine (hard to test until we release a PyPI package for it; 3.19 should be out very soon, though, @jcfr is working on it)